unpackaged/config/trial/standardValueSetTranslations/ContactRole-de.standardValueSetTranslation

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<StandardValueSetTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
Maybe these standardValueSetTranslations should live in the trial_translations folder? There's some duplicated metadata between the two.